packages/playground/optimize-missing-deps/package.json

```diff
@@ -7,7 +7,8 @@
     "postinstall": "ts-node ../../../scripts/patchFileDeps.ts"
   },
   "dependencies": {
-    "missing-dep": "file:./missing-dep"
+    "missing-dep": "file:./missing-dep",
+    "multi-entry-dep": "file:./multi-entry-dep"
```

Comment:
This is a dependency of `missing-dep` so we dont need to add it here. (It's my negligent when building the test🙈)

Reply:
done